shell/src/main/java/alluxio/cli/fs/command/GetSyncPathListCommand.java

```diff
@@ -41,10 +43,23 @@ public String getCommandName() {
 
   @Override
   public int run(CommandLine cl) throws AlluxioException, IOException {
-    List<String> files = mFileSystem.getSyncPathList();
+    List<SyncPointInfo> files = mFileSystem.getSyncPathList();
     System.out.println("The following paths are under active sync");
-    for (String file : files) {
-      System.out.println(file);
+    for (SyncPointInfo syncPointInfo : files) {
+      System.out.print(syncPointInfo.getSyncPointUri() + "\t");
+      switch (syncPointInfo.getSyncStatus()) {
+        case NOT_INITIALLY_SYNCED:
+          System.out.println("NEVER SYNCED");
```

Comment:
I think all of these user facing statuses should refer to initial sync somehow. Basically, it seems like initial sync can be "skipped", "in progress", or "done".